services/github/gist/github-gist-last-commit-redirect.tester.js

```diff
@@ -1,6 +1,7 @@
 import { ServiceTester } from '../../tester.js'
+
 export const t = new ServiceTester({
-  id: 'GithubGistLastCommitRedirect',
+  id: 'GistLastCommitRedirect',
```

Comment:
Annoyingly, the built-in `GITHUB_TOKEN` secret has enough permissions to run _neeeeearly_ all of the GitHub test suite. The one exception to this is Gists. Also it is not possible to grant a token a `gist: read` scope: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token This is a limitation which does apply to github actions/workflow tokens but doesn't exist for PATs.

The way I've proposed to deal with this is we rename the `GithubGist` service to just `Gist`. This will mean if I submit a PR

- "do a thing to [github] services" then all the tests should run and pass, even if the PR is submitted from a fork
- "do a thing to [gist] services" then all the tests should run and pass from a local branch but will fail from a fork

The downside of that is we have to remember that `GithubGist` has a different service tag from `Github`. The other thing we could do is leave it and accept that a handful of tests will fail if we try to run the full github suite from a forked PR.